app/forms/idv/phone_form.rb

```diff
@@ -38,22 +39,29 @@ def submit(params)
       FormResponse.new(success: success, errors: errors, extra: extra_analytics_attributes)
     end
 
-    def user_has_multiple_phones?
-      @user.phone_configurations.many?
-    end
-
     private
 
     attr_writer :phone, :otp_delivery_preference
 
-    def initial_phone_value(input_phone)
-      initial_phone = input_phone
-      initial_phone ||= begin
-        user_phone = MfaContext.new(user).phone_configurations.take&.phone
-        user_phone if valid_phone?(user_phone, phone_confirmed: true)
+    # @return [Array<string,string>] The international_code and phone values to use.
+    def determine_initial_values(international_code: nil, phone: nil, **_)
```

Comment:
Previously, we were initializing the `phone` field but not the `international_code` field. This could lead to cases where non-US phone numbers would end up not passing client-side validation, since the `PhoneInputComponent` would attempt to parse them as with a `US` country code.